### PR TITLE
Disable Dynamic Log File

### PR DESCRIPTION
**Pull request by yinggeh**
Disable dynamic log file location, aka `--log_file` API.

### CODE REVIEW

File: src/grpc/grpc_server.cc [hunk 1]
```diff
@@ -1463,32 +1463,10 @@ CommonHandler::RegisterLogging()
...
-            // to the global object.
-            err = TRITONSERVER_ServerOptionsSetLogFile(
-                nullptr, log_file_path.c_str());
-            if (err != nullptr) {
-              GOTO_IF_ERR(err, earlyexit);
-            }
-          }
+          err = TRITONSERVER_ErrorNew(
+              TRITONSERVER_ERROR_UNSUPPORTED,
+              "updating log file was deprecated and no longer supported");
```
Comment (nnshah1) [inline src/grpc/grpc_server.cc:1468]: would continue to use the variable `setting_name` for completeness

Comment (nnshah1) [reply src/grpc/grpc_server.cc:1468]:
```suggestion
              "log file location can not be updated through network protocol");
```

**Review by nnshah1 — APPROVED** [review]

File: src/grpc/grpc_server.cc (hunk 1 above)
Comment (nnshah1) [reply src/grpc/grpc_server.cc]: little tricky - given that it's the only one probably clear enough from context. 

Was thinking:
```
 setting_name + " can not be updated through newtork protocol"  
```

Comment (nnshah1) [reply src/grpc/grpc_server.cc]: @Tabrizian - any thoughts one way or the other?

Comment (Tabrizian) [reply src/grpc/grpc_server.cc]: Error message looks good to me.

**Pull request merged**